src/layers/legacy/medCoreLegacy/gui/toolboxes/medRegistrationSelectorToolBox.cpp

```diff
@@ -126,6 +126,9 @@ void medRegistrationSelectorToolBox::changeCurrentToolBox(int index)
     connect (currentToolBox(), SIGNAL (success()),this,SLOT(enableSelectorToolBox()));
     connect (currentToolBox(), SIGNAL (failure()),this,SLOT(enableSelectorToolBox()));
 
+    unsigned int index = this->comboBox()->currentIndex();
```

Comment:
index creates a compilation error since it's the name of a parameter of this function. I'm going to do PRs to solve that